id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/CredentialRequestGeneratorBootApplication.java

```diff
@@ -0,0 +1,44 @@
+package io.mosip.credential.request.generator;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.FilterType;
+import org.springframework.context.annotation.Import;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+import io.mosip.credential.request.generator.api.config.CredentialRequestGeneratorConfig;
+import io.mosip.credential.request.generator.entity.CredentialEntity;
+import io.mosip.idrepository.core.config.IdRepoDataSourceConfig;
+import io.mosip.idrepository.core.helper.RestHelper;
+import io.mosip.idrepository.core.security.IdRepoSecurityManager;
+import io.mosip.idrepository.core.util.DummyPartnerCheckUtil;
+import io.mosip.kernel.dataaccess.hibernate.config.HibernateDaoConfig;
+import io.mosip.kernel.dataaccess.hibernate.repository.impl.HibernateRepositoryImpl;
+
+/**
+ * The Class CredentialRequestGeneratorApp.
+ *
+ */
+@SpringBootApplication
+@Import(value = { java.lang.String.class, DummyPartnerCheckUtil.class, RestHelper.class, IdRepoSecurityManager.class,
+		CredentialRequestGeneratorConfig.class })
+@ComponentScan(basePackages = { "io.mosip.*", "${mosip.auth.adapter.impl.basepackage}" }, excludeFilters = {

```

Comment:
please scan specific libraries that required

Reply:
DOne.